### ISSUE TITLE
webwormhole.io web interface non-functional on Chrom{e,ium} 85.0.4183.39 and 84.0.4147.89

### ISSUE THREAD

**Issue by mmirg** [saljam/webwormhole#66]
The webwormhole.io web interface appears to not work properly on recent versions of Chrom{e,ium}. Neither I nor the recipient of my file could get it to work on recent versions of the browser. It fails for me on chromium 85.0.4183.39 (current beta) and simply shows a "loading" button and will not accept a passphrase. The same happened on my recipient's end with Chrome 84.0.4147.89 (current stable).

Here's the error that's thrown in the console:
```
main.js:447 Uncaught (in promise) TypeError: chrome.runtime.getURL is not a function
    at browserhacks (main.js:447)
    at main.js:485
    at main.js:520
```

The web interface works without a problem in Firefox (I tested on 72.0.2) and also from the command line (although the command line installation instructions do not work on either GNU/Linux or MacOS without exporting `GO111MODULE=on`.

**Comment by saljam** [comment]
thanks for reporting! looks like the web extension work broke it in chrome. i'm rolling it back now.

**Comment by saljam** [comment]
fixed in 6f87ce7 and redeployed. works for me in chrome now.
